LondonBritishLibrary/orient/BLorient12996.xml

```diff
@@ -0,0 +1,292 @@
+                                    <note>On pronouns.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.5">
+                                    <locus from="10r"/>
+                                    <title ref="LIT00000000000000000000000#VerbalAffixes">De Affixis Verborum</title>
+                                    <note>On verbal affixes.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.6">
+                                    <locus from="11r"/>
+                                    <title ref="LIT00000000000000000000000#Oration2">De alijs Orationis partibus</title>
+                                    <note>Second part of orations as language samples.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.7">
+                                    <locus target="#12r"/>
+                                    <title ref="LIT00000000000000000000000#Numbers">De Numero</title>
+                                    <note>On the numbers (i.e. <foreign xml:lang="am">፩</foreign>, <foreign xml:lang="am">፪</foreign>, 
+                                        <foreign xml:lang="am">፫</foreign>, etc.), which do not have the conventional two horizontal dashes.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.8">
+                                    <locus from="12r"/>
+                                    <title ref="LIT00000000000000000000000#Conjugation">De Verbo, et ejus Conjugatione</title>
+                                    <note>On the verb and its conjugation.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.9">
+                                    <locus from="14v"/>
+                                    <title ref="LIT00000000000000000000000#AffirmativeVerbs">Conjugatio Verbi Affirmativi</title>
+                                    <note>Conjugation of affirmative verbs, that are described here as <foreign xml:lang="it">Ingiuriare</foreign> 
+                                        (i.e. verbs 'to offend').</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.10">
+                                    <locus from="15v"/>
+                                    <title ref="LIT00000000000000000000000#Particles">Modus et forma struendi reliquos Modos</title>
+                                    <note>Description containing twenty-four noteson the different particles used with the verb.</note>
+                                </msItem>
+                                <msItem xml:id="p1_i1.11">
+                                    <locus from="21r"/>
+                                    <title ref="LIT00000000000000000000000#Participle">De Participio sive Supino in Rus</title>
```

Comment:
Maybe make clear that "Rus." is an abbreviation by including the dot?

Reply:
This was not clear to me.